test/unittest/dds/participant/ParticipantTests.cpp

```diff
@@ -36,16 +44,23 @@
 #include <fastdds/dds/subscriber/Subscriber.hpp>
 #include <fastdds/dds/topic/qos/TopicQos.hpp>
 #include <fastdds/rtps/attributes/RTPSParticipantAttributes.h>
+#include <fastdds/rtps/attributes/ServerAttributes.h>
+#include <fastdds/rtps/common/Locator.h>
+#include <fastdds/rtps/participant/RTPSParticipant.h>
```

Comment:
It is not being used. Good catch!

Reply:
It is necessary because `RTPSParticipant::getRTPSParticipantAttributes` is defined within that header file. Without this include, the project does not build.